src/psyclone/tests/test_files/dynamo0p3/infrastructure/argument_mod.F90

```diff
@@ -63,15 +63,17 @@ module argument_mod
   integer, public, parameter :: GH_MAX       = 17
 
   ! Function-space labels
-  integer, public, parameter :: W0      = 100
-  integer, public, parameter :: W1      = 101
-  integer, public, parameter :: W2      = 102
-  integer, public, parameter :: W3      = 103
-  integer, public, parameter :: Wtheta  = 104
-  integer, public, parameter :: W2V     = 105
-  integer, public, parameter :: W2H     = 106
-  integer, public, parameter :: Wchi    = 107
-  integer, public, parameter :: ANY_W2  = 108
+  integer, public, parameter :: W0        = 100
+  integer, public, parameter :: W1        = 101
+  integer, public, parameter :: W2        = 102
+  integer, public, parameter :: W2V       = 103
+  integer, public, parameter :: W2H       = 104
+  integer, public, parameter :: W2broken  = 105
+  integer, public, parameter :: W2trace   = 106
+  integer, public, parameter :: W3        = 107
+  integer, public, parameter :: Wtheta    = 108
+  integer, public, parameter :: Wchi      = 109
```

Comment:
Out of interest, why do we not support Wchi? I know it's been there for a while!

Reply:
Very good question! The short answer is that I do not really know (it is on LFRic trunk but not actually used) so I opened issue #539 - hopefully Tom M. will be able to provide an answer.

Reply:
Thanks for raising this with @thomasmelvin.